Visual tests are failing on R-devel

(originally found at https://github.com/tidyverse/ggplot2/pull/4115#issuecomment-653655613)

``` 
 ── 1. Error: aesthetics are drawn correctly (@test-aes.r#160)  ─────────────────
Graphics API version mismatch
Backtrace:
 1. vdiffr::expect_doppelganger(...)
##[error]Error: R CMD check found ERRORs
Execution halted
 2. vdiffr:::writer(fig, testcase, title)
 3. vdiffr:::svglite(file, user_fonts = get_aliases())
 4. vdiffr:::svglite_(...)
```
(https://github.com/tidyverse/ggplot2/pull/4115/checks?check_run_id=835351166#step:10:142)

--- a/.github/workflows/R-CMD-check.yaml
+++ b/.github/workflows/R-CMD-check.yaml
@@ -10,7 +10,7 @@ name: R-CMD-check
 
 # Increment this version when we want to clear cache
 env:
-  cache-version: v2
+  cache-version: v3
 
 jobs:
   R-CMD-check:

